carbon-design-system/carbon
# fix(components,themes): move theme fallback to themes package

Closes #8838 

This PR moves our logic for adding fallback theme token values from the components package to the themes package.

#### Changelog

**New**

**Changed**

- Update themes generated mixin step to include component token mappings for new tokens being used in components

**Removed**

- Removed token mappings from components package

#### Testing / Reviewing

- Run the storybook and verify that the token values are correct in the component
- View the netlify preview and make sure custom properties work as expected
